srtcore/channel.cpp

```diff
@@ -502,8 +522,8 @@ int CChannel::sendto(const sockaddr* addr, CPacket& packet) const
       const int res = ::sendmsg(m_iSocket, &mh, 0);
    #else
       DWORD size = (DWORD) (CPacket::HDR_SIZE + packet.getLength());
-      int addrsize = m_iSockAddrSize;
-      int res = ::WSASendTo(m_iSocket, (LPWSABUF)packet.m_PacketVector, 2, &size, 0, addr, addrsize, NULL, NULL);
+      int addrsize = addr.size();
+      int res = ::WSASendTo(m_iSocket, (LPWSABUF)packet.m_PacketVector, 2, &size, 0, &addr, addrsize, NULL, NULL);
       res = (0 == res) ? size : -1;
```

Comment:
`const int res`

Reply:
How will you then rewrite a new value to `res` in the next line?

Reply:
```
const int sendres = ::WSASendTo(...
const int res = (0 == sendres) ? size : -1;
```
Although may stay as it is.